thoth/storages/graph/models_performance.py

```diff
@@ -37,8 +37,8 @@
 class PerformanceIndicatorBase:
     """A base class for implementing performance indicators."""
 
-    # ML framework used for the performance indicator.
-    framework = Column(String(256), nullable=True)
+    # library used for the performance indicator.
+    library = Column(String(256), nullable=True)
```

Comment:
Not sure with `library` naming here - we have also Python specific performance indicators.

Reply:
What about `module`?

Reply:
I think that's better. What about `component`?

Reply:
It is also good, I will change it in PI as well :)